docs/extending/customising_group_views.md

```diff
@@ -106,3 +106,27 @@ INSTALLED_APPS = [
     ...,
 ]
 ```
+
+## Customising the Group Editor Object Permissions Order
+
+The order that object types appear in the Group Editor "Object permissions" and "Other permissions" sections can be configured by registering that order in one or more AppConfig definitions. The order value is typically an integer between 0 and 999, although this is not enforced.
```

Comment:
```suggestion
The order that object types appear in the group's "Object permissions" and "Other permissions" sections can be configured by registering that order in one or more `AppConfig` definitions. The order value is typically an integer between 0 and 999, although this is not enforced.
```

Just avoiding the 'Group Editor' term, we do not really use that in this page.

Reply:
Changed in [1c515e4](https://github.com/wagtail/wagtail/pull/10469/commits/1c515e4d49262d2a8ed80eb201fa8046670563be)